src/stories/roundCheckbox.stories.js

```diff
@@ -0,0 +1,24 @@
+import { createRoundCheckbox } from "./roundCheckbox";
+
+export default {
+    title: "Component/Round Checkbox",
```

Comment:
I think that folder Component/Checkbox  is better/ We will have there your round component and others checkbox